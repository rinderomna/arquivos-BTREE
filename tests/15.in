8 tipo1 binario15.bin indice15.bin 1
1 id 568
1 id 868
