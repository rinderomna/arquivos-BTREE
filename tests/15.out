24465.390000
1276.700000
